.github/ISSUE_TEMPLATE/project-suggestion.md

```diff
@@ -0,0 +1,59 @@
+---
+name: Project Proposal / Suggestion
+about: Idea for a new project that will improve or enhance the TAG Environmental Sustainability (If you want to implement the idea personally, name it "Proposal" if NOT call it "Suggestion").
+title: "[Project Proposal / Suggestion] some descriptive title"
+labels: "triage-required"
+assignees: ''
+---
+
+<!-- Thank you for opening this proposal! If you personally want to lead the project, call it a "proposal", IF NOT, call it a "suggestion". If you want to lead the project, it is much more likely that the proposal will be accepted! -->
+
+## Project details
+
+### Description
+<!-- describe your idea here -->
+
+...
+
+### Impact
+<!-- Describe your hopes for how this would reduce risk for the cloud native ecosystem. Who will this help? How will it help them? -->
+
+...
+
+### Scope
+<!-- How much effort will this take? ok to provide a range of options if or "not yet determined"  -->
+
+...
+
+### Goals and Non Goals
+<!-- How much effort will this take? ok to provide a range of options if or "not yet determined"  -->
+
+...
+
+### Deliverables
+<!-- What artifacts are created to complete this project? Blogs, whitepaper, documentation, video, in person meeting, etc. -->
+
+- *Deliverable A*
+- *Deliverable B*
+
+### Additional info and comments
+
+- Reference to supporting material: `NONE`
+- Links to related site: `NONE`
+- Feel free to delete this section if you don't have more info: `NONE`
+
+## TODO
+
+Please complete these steps in order.
```

Comment:
I'd remove the "in order".  Sometimes, did instance, I'm doing leads and proposal concurrently.